.github/workflows/pr-build.yml

```
@@ -0,0 +1,52 @@
+name: PR 빌드 체크
+
+on:
+  pull_request:
+    branches: ["main", "dev"]
+
+jobs:
+  name: build (pull_request)
```

actionlint (1.7.7)

"steps" section is missing in job "name"

(syntax-check)